app/validation/harvest_grow.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any, Dict, List, Tuple, Optional
+from collections import Counter
+
+from app_config import AppConfig
+
+from app.models.grow import Grow
+from app.models.grow_phase import GrowPhase
+
+
+def validate_harvest_grow(app_config: AppConfig, message) -> Tuple[bool, str]:
```

Comment:
Add a description for the function. You can check the conventions described here https://stackoverflow.com/questions/32566466/multi-line-description-of-a-parameter-description-in-python-docstring/32866294